docs/overview/components/conductor.md

```diff
@@ -0,0 +1,65 @@
+# The Astria Conductor
+
+The Conductor can be thought of as the consensus implementation of a rollup full
+node, similar to op-node within the OP Stack. The conductor is run as one half
+on a rollup full node, where the other half is the execution engine. Its role is
+to connect the sequencer and DA layers to the rollup execution layer by deriving
+the relevant rollup transactions for each block and passing them to the
+execution layer.
+
+The conductor’s flow looks like:
+![Astria Conductor](../assets/astria-conductor.png)
```

Comment:
I'd write out DA Layer in the big orange box to the right

Reply:
removing this diagram for now